package.json

```diff
@@ -92,7 +92,17 @@
                         "B"
                     ],
                     "default": "GB"
-                }
+                },
+				"resmon.alignLeft": {
```

Comment:
Is it just on my screen that there're alignment issues with the JSON itself?

I think I'll just bring it in then I'll fix the alignment on my next iteration, unless you want to.

I also want to change this to an enum from a boolean to accommodate future values (maybe centered alignment?) or maybe we will have more locations that this information can be displayed.

Reply:
I'm totally fine with that.